VALUES_AND_PRIORITIZAION.md

```diff
@@ -0,0 +1,48 @@
+# Values and Priorities
+
+> Document Status: Proposal, Work In Progress
+
+## Context
+
+Define the `Technical` values shared by the project and their relatively priority
+that are applied when making tradeoffs.
+
+
+## Values and priority level
+
+This is only a subset of the overall prioties, listing only those which are at the
+top of the priority list. Anything on this list is very important for the project
+despite the relative priorities shown.
+
+- Prio 1 - Developer experience
+- Prio 2 - Stability
+- Prio 3 - Operational qualities
+- Prio 4 - Node.js maintainer experience
+- Prio 5 - Technology and API Currency
+
+## Value descriptions
+
+### Developer Experience
+  - Approachability (both technical and community)
+  - Great Documentation 
```

Comment:
```suggestion
We value ensuring that developers are productive and enjoy developing with Node.js. Some key elements of this include:
  - Approach-ability (both technical and community)
  - Great Documentation 
  - Bundling API/components which reduce the friction of using them, even though they could be external
  - Enabling/supporting external packages to ensure overall developer experience
```
  


  

  - bundling things to remove obstacles.
  - Approach-ability (both technical and community)
  - Good API and backwards compatibility support for external tooling (webpack, react, etc.), listening for feedback.
  - Tooling (webpack, react, 
  - Great Documentation 
  
```